packages/react/src/components/DateTimePicker/DateTimePickerV2.test.jsx

```diff
@@ -607,8 +607,8 @@ describe('DateTimePickerV2', () => {
         }}
       />
     );
-    // default value is YYYY-MM-DD hh:mm A
-    expect(screen.getByText('YYYY-MM-DD hh:mm A')).toBeVisible();
+    // default value is YYYY-MM-DD hh:mm XM
+    expect(screen.getByText('YYYY-MM-DD hh:mm XM')).toBeVisible();
```

Comment:
I don't understand.  Why would expect the datetime mask to be XM ?  That's not even a valid dayjs mask (at least not as used)

Reply:
@sls-ca  Hello Sean, this is an issue reported in Graphite that placeholder texts were inconsistent. Like in the screenshot, time picker follows carbon guideline to use 'XM' as the meridian and date picker uses the time format. User reported it was confusing to see the differences. And 'A' is not as intuitive as 'XM' described in the guideline. To make it consistent, we made sure dateTimePicker use 'XM' in the placeholder text.

![image](https://user-images.githubusercontent.com/24279794/236228955-d1c54318-f00b-4fb8-9d4d-ebb881644ef2.png)